### ISSUE TITLE
VNC server errors

### ISSUE THREAD

**Issue by Protektor-Desura** [jgoerzen/docker-bbs-renegade#2]
I am not able to get in to the BBS through the web VNC interface. I get errors. I was able to get in via telnet but I couldn't set up account because it wanted me to create a new account then the sysop would approve it but I couldn't login as the sysop to approve the account.

**Comment by jgoerzen** [comment]
Without knowing what the errors are, it's impossible to help.

**Comment by Protektor-Desura** [comment]
I'm guessing it isn't a web VNC like guacamole but instead you need a VNC client to connect?

**Comment by jgoerzen** [comment]
Correct.  Raw VNC.  If you want a web interface, you can stand that up in a separate container/service.

**Comment by mro-1337** [comment]
Hello, please do not run this dockercontainer.   it uses a version of renegade that has custom tweaks that aren't compatible with how this user setup rg in docker. 
thanks,

-- renegadebbs.net domain owner
